### DISCUSSION TITLE
GitHub Copilot for Business

### DISCUSSION THREAD

**Discussion by RitchieFresh** [community/community#140234] [Copilot Conversations]
I am using VS Code to access GitHub Copilot for Business. 

I was able to access Copilot in VS Code and create a Python file. The thing is I logged out of my computer and when I logged back in I was not able to access Copilot using VS Code; even though, I was able to use Copilot an hour before it stopped working. 

In VS Code, I clicked the Settings icon at the bottom right of the screen. in VS Code bottom right corner, it says Signing in to github.com...., after a few minutes, in the web browser window that pops up I login and after a minute I am informed by VS Code that You have not yet finished authorizing this extension to use GitHUb. Would you like to try a different way?  

Source: GitHub Authentication Yes \ No

I click Yes and this bring my back to the browser window where Github is unable to login

However, when I open the GitHub Copilot extension, when I click on GitHub.com, the browser opens up my account where I can toggle between my individual and organizational accounts.

I am stumped ... and not sure I will get any assistance from anyone associated with Copilot and or Github; therefore, I am requesting assistance from the community.

Thanks in advance for any assistance.

Best regards to the community,
Rich

**Comment by feijoes** [comment]
Try uninstalling the extensions again and logging out of GitHub. Open VS Code and press Ctrl + Shift + P to open the Command Palette. Type GitHub: Sign Out and press Enter

**Comment by RitchieFresh** [comment]
I tried to feijoes recommendation 

This is the response I got when I tried to login, "You have not yet finished authorizing this extension to use GitHub. Would you like to try a different way? (local server)"

I don't know what to do next.

This is crazy because I was able to use the program for about a day before this happened.

**Reply by feijoes** [reply]
I found this Stack Overflow question with the same problem. Try one of the solutions: https://stackoverflow.com/questions/78066767/cannot-sign-in-with-github-in-vscode.

If that doesn't work, check this: https://github.com/github/vscode-github-actions/issues/77#issuecomment-1503960028.

**Comment by RitchieFresh** [comment]
I am going to follow the step provided in If that doesn't work, check this: https://github.com/github/vscode-github-actions/issues/77#issuecomment-1503960028.

I think somehow there's two credentials at play and for some reason I don't have both same in windows password manager. It is possible Windows retained one password and overwrite the second password. 

I have Github [RitchieFresh] and Github Copilot for Business [Vuniabu]. In Coplit for Business, in the organisation setting I assigned Copilot access to GitHub [RitchieFresh].

I think windows, overwrite the password for Github[RitchieFresh} with credentials for Github Copilot for Business[Vuniabu].

At one point, when I was trying to sign into VS Code Copilot, it had to login credentials: one worked but the second one didn't, so I deleted the one that did work. I was able to continue to use the one that worked until finally it stopped working. 

I have to run for a moment, later today I will go the information you sent me. 

Thank you for everything you are doing. I truly appreciate it. Thank you.

**Comment by RitchieFresh** [comment]
![image](https://github.com/user-attachments/assets/a21ab9cb-1792-4fe8-b3e4-2a3f708d56a7)

**Comment by RitchieFresh** [comment]
![image](https://github.com/user-attachments/assets/464d472c-1386-4c0a-a98a-b88d075dd3aa)

**Comment by RitchieFresh** [comment]
Now this is the message I am getting when I try to sign in 

![image](https://github.com/user-attachments/assets/25a3eb97-a175-4bb3-99ce-338c9fe8ebb5)

When I click yes, I am taken to my browser where I am ask to login, I click continue and am brought back to the above image

**Comment by liukai2019** [comment]
I tried several methods I searched on Stack Overflow and GitHub. Basically, they all don't work. And I am quite certain I didn't change anything. But somehow, I successfully logged in. I don't know what happened.
